gatsby-plugin-theme-ui/README.md

```diff
@@ -0,0 +1,27 @@
+
+# gatsby-plugin-theme-ui
+
+Adds the Theme UI ThemeProvider context to Gatsby themes and sites, deeply merging them into a single context.
+
+```sh
+npm i gatsby-plugin-theme-ui
+```
+
+```js
+// gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: 'gatsby-plugin-theme-ui',
+      options: {
+        // required
+        theme: require.resolve('./src/theme.js'),
+        // optional
+        components: require.resolve('./src/components.js'),
+        // optional, initial color mode
+        colorMode: 'light',
```

Comment:
`defaultColorMode` or something seems more obvious. It's also possible that this color mode doesn't exist if someone overrides the tokens right?

Reply:
Renamed to match the prop name `initialColorMode` – if a color mode object isn't defined in the `theme.colors`, it will default to whatever is in `theme.colors` – a Gatsby theme author could enable this option and an end-user could opt-out of using color modes by not defining them in the theme object, and I guess it could make sense to allow the final instance of this plugin (e.g. the end users site) to disable the color mode state entirely